invokeai/app/api/routers/model_records.py

```diff
@@ -162,3 +171,136 @@ async def add_model_record(
 
     # now fetch it out
     return record_store.get_model(config.key)
+
+
+@model_records_router.post(
+    "/import",
+    operation_id="import_model_record",
+    responses={
+        201: {"description": "The model imported successfully"},
+        404: {"description": "The model could not be found"},
+        415: {"description": "Unrecognized file/folder format"},
+        424: {"description": "The model appeared to import successfully, but could not be found in the model manager"},
```

Comment:
In the circumstance that the install job runs to completion but the subsequent attempt to fetch the config record fails, we now return status code 424 rather than 404. The 404 error has been removed.